src/components/Footer.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import { Col, Row, Container } from 'reactstrap';
+
+import Logo from './Logo';
+
+const Wrapper = styled.div`
+  background-color: ${props => props.theme.colors.blue};
+  color: #fff;
+  padding-top: 4em;
+  height: 15em;
+`;
+
+const OptionsContainer = styled(Container)`
```

Comment:
There's an option `fluid` which can be passed to container which acts as full-width